packages/core/src/client.ts

```diff
@@ -666,6 +666,10 @@ export class MedplumClient extends EventTarget {
       this.refreshProfile().catch(console.log);
     }
 
+    if (options?.accessToken) {
+      this.accessToken = options.accessToken;
```

Comment:
We probably want to use `this.setAccessToken(options.accessToken)` here to ensure all the client state is set correctly

Reply:
Good catch